src/ngraph/runtime/intelgpu/unit_test.manifest

```diff
@@ -79,3 +79,6 @@ scatter_add_scalar_indices
 scatter_nd_add_batch_2d_to_3d
 scatter_nd_add_2d_to_3d
 zero_sized_erf
+fake_quantize
+fake_quantize_with_clip
+fake_quantize_with_clip_across_channels
```

Comment:
These tests might work in case of decomposed FakeQuantize

Reply:
At this moment this PR lacks operator implementation, thus I'll leave these uts skipped.